src/containers/DataPackageView.js

```diff
@@ -140,21 +145,30 @@ export class DataPackageViewContainer extends React.Component {
 
   render() {
     return (
-      <DataPackagePanel specs={this.state.specs}/>
+      <div className="col-lg-10">
+        <DataPackagePanel specs={this.state.specs}/>
+        <ReactMarkdown source={this.props.metadata.readme}/>
```

Comment:
Have to find a better name. This attribute will hole ```readme and other stuff (e.g title, description)``` , only read me will not be enough.